checkov/common/util/json_utils.py

```diff
@@ -0,0 +1,9 @@
+import json
+
+
+class CustomJSONEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, set):
+            return json.dumps(list(o))
```

Comment:
I tried

`json.JSONEncoder.default(self, list(o))`

here, and I got "Object of type list is not JSON serializable". Not sure why.

Reply:
I think you can just change ti to
```python
return list(o)
```
for further info https://docs.python.org/3/library/json.html#json.JSONEncoder.default